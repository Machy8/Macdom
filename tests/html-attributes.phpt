<?php

use Tester\Assert;


require __DIR__ . '/bootstrap.php';

Assert::matchFile(
	EXPECTED . "/html-attributes.html",
	$macdom->compile(file_get_contents(ACTUAL . "/html-attributes.html"))
);
