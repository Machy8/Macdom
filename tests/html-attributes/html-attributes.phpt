<?php

use Tester\Assert;

require '../bootstrap.php';

$tested = file_get_contents("../test-templates/html-attributes-a.html");
$result = file_get_contents("../test-templates/html-attributes-b.html");

Assert::same($result, $o->compileContent($tested));
