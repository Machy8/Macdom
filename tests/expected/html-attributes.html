<div id="id" class="class" data-label=label></div>
<div data-mix=;+=´´)ú¨§ů-.,-*/+_:\\()[]@&|\# id=mix class="class"></div>
<input onclick='alert("Some text here 123 456 789")' onmouseover="alert('Some text here 123 456 789')">
<input data-label=someLabel data-second-label="Some label" data-third-label='Some label'>